pkg/client/parse.go

```diff
@@ -0,0 +1,156 @@
+package client
+
+import (
+	"encoding/xml"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/guidewire-oss/fern-junit-client/pkg/models/fern"
+	"github.com/guidewire-oss/fern-junit-client/pkg/models/junit"
+)
+
+func parseReports(testRun *fern.TestRun, filePattern string, verbose bool) error {
+	files, err := filepath.Glob(filePattern)
```

Comment:
so i see it can take a file or a set of files but the full path of where the file is still needed correct ?

Reply:
`Glob` returns a list of file paths after parsing the given pattern.
For example, if you give it `*.xml`, it will return the full file paths of every `.xml` file in the directory.